content/organizations/managing-peoples-access-to-your-organization-with-roles/adding-a-billing-manager-to-your-organization.md

```diff
@@ -49,6 +49,12 @@ Billing managers **are not** able to:
 
 ## Inviting a billing manager
 
+{% note %}
+
+**Note:** If your organization is managed using [Enterprise Accounts](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts) you will not be able to invite Billing Managers at the Organization level. 
```

Comment:
```suggestion
**Note:** If your organization is managed using [Enterprise Accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts) you will not be able to invite Billing Managers at the Organization level. 
```

Reply:
Perfect fix! I'm going to commit your suggestion so that the checks pass.